doc/sphinx/language/core/records.rst

```diff
@@ -235,34 +235,12 @@ the errors of inductive definitions, as described in Section
 Primitive Projections
 ~~~~~~~~~~~~~~~~~~~~~
 
-.. flag:: Primitive Projections
-
-   This :term:`flag` turns on the use of primitive
-   projections when defining subsequent records (even through the ``Inductive``
-   and ``CoInductive`` commands). Primitive projections
-   extended the Calculus of Inductive Constructions with a new binary
-   term constructor `r.(p)` representing a primitive projection `p` applied
-   to a record object `r` (i.e., primitive projections are always applied).
-   Even if the record type has parameters, these do not appear
-   in the internal representation of
-   applications of the projection, considerably reducing the sizes of
-   terms when manipulating parameterized records and type checking time.
-   On the user level, primitive projections can be used as a replacement
-   for the usual defined ones, although there are a few notable differences.
-
-.. flag:: Printing Primitive Projection Parameters
-
-   This compatibility :term:`flag` reconstructs internally omitted parameters at
-   printing time (even though they are absent in the actual AST manipulated
-   by the kernel).
-
-Primitive Record Types
-++++++++++++++++++++++
-
-When the :flag:`Primitive Projections` flag is on, definitions of
-record types change meaning. When a type is declared with primitive
-projections, its :g:`match` construct is disabled (see :ref:`primitive_projections` though).
-To eliminate the (co-)inductive type, one must use its defined primitive projections.
+When the :flag:`Primitive Projections` flag is on or the 
+:attr:`projections(primitive)` is supplied, definitions of record types change 
+meaning. When a type is declared with primitive projections, its :g:`match` 
+construct is disabled (see :ref:`primitive_projections` though).
+To eliminate the (co-)inductive type, one must use its defined primitive 
+projections.
 
 .. The following paragraph is quite redundant with what is above
```

Comment:
Do you still consider this comment to be relevant? It seems to me that it could be removed after your changes.

Reply:
I will keep it as I think it's useful when skimming the documentation.